src/workerd/api/crypto/impl.h

```diff
@@ -293,6 +294,14 @@ const SslDisposer<T, sslFree> SslDisposer<T, sslFree>::INSTANCE;
 // Using BN_clear_free here ensures that any potentially sensitive information in the
 // BIGNUM is also cleansed when it is freed.
 
+using UniqueBignum = std::unique_ptr<BIGNUM, void(*)(BIGNUM*)>;
+kj::Maybe<kj::Own<BIGNUM>> toBignum(kj::ArrayPtr<const kj::byte> data);
```

Comment:
stylistical suggestion: ArrayPtr/Array have `.as()` method that is designed to help with things like `ptr.as<Bignum>()`. I found it very helpful for Rust conversions and makes code much easier to read.

Reply:
Perhaps if we introduce a `Bignum` helper that wraps `BIGNUM` at some point. Given that this is mostly just shifting around the way the existing code already worked, I'd prefer to keep it as is for now but I'm going to be continuing to iterate on this area.